libsql/internal/ws/driver.go

```diff
@@ -86,18 +86,26 @@ func convertToNamed(args []driver.Value) []driver.NamedValue {
 }
 
 func (s stmt) Exec(args []driver.Value) (driver.Result, error) {
-	ctx, cancel := context.WithTimeout(context.Background(), defaultWSTimeout)
-	defer cancel()
-	return s.c.ExecContext(ctx, s.query, convertToNamed(args))
+	return s.ExecContext(context.Background(), convertToNamed(args))
 }
 
 func (s stmt) Query(args []driver.Value) (driver.Rows, error) {
-	ctx, cancel := context.WithTimeout(context.Background(), defaultWSTimeout)
-	defer cancel()
-	return s.c.QueryContext(ctx, s.query, convertToNamed(args))
```

Comment:
Why not `return s.QueryContext(context.Background(), args)`? `QueryContext` will call `convertToNamed` internally, no?

Reply:
(this is same like ExecContext, explained in other comment)